lms/djangoapps/course_blocks/api.py

```diff
@@ -24,6 +22,7 @@
     start_date.StartDateTransformer(),
     user_partitions.UserPartitionTransformer(),
     visibility.VisibilityTransformer(),
+    hidden_content.HiddenContentTransformer(),
```

Comment:
Note to self: Thanks to @efischer19 for pointing out that the grades path needs to exclude this transformer.